tests/integration/states/pip.py

```diff
@@ -361,7 +364,6 @@ def test_issue_6833_pip_upgrade_pip(self):
             # previously installed
             ret = self.run_function(
                 'pip.install', ['pip==6.0'], upgrade=True,
-                ignore_installed=True,
```

Comment:
@terminalmage The `ignore_installed` option was creating a corrupted pip environment on OSX. Pip commands would stack trace.